Enrise/accessibility-checker-infrastructure
# Sac 24 - Made it a proper project with testing in the pipeline

# What

In this project everything we had is turned into a proper (example) project with accessibility checking in the pipeline.

# How to test

1. run `make init` & `make build`
1. checkout http://localhost:1000
1. run `make test` to see that testing as developer works (docker + local)
1. check if the test in the pipeline works
